docs/IE_PLUGIN_DG/NewTransformation.md

```diff
@@ -139,10 +177,12 @@ This example show how we can construct pattern when operation has arbitrary numb
 
 @snippet example_ngraph_utils.cpp pattern:concat_example
 
-This example shows how to use predicate to construct pattern where operation has two different types.
+This example shows how to use predicate to construct pattern where operation has two different types. Also it shows how to match pattern manually on given node.
 
 @snippet example_ngraph_utils.cpp pattern:predicate_example
 
+> **Note**: be careful with manual matching because Matcher object holds matched nodes. To clear match use m->clear_state() method.
+
 TODO: add examples for ngraph::pattern::op::Any
```

Comment:
should this `TODO` be fixed in this PR?

Reply:
I removed TODOs but I'll finalyze this document in other PR.